### PR TITLE
docs(multi-tenant): add multi tenant doc to how to guides

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -322,6 +322,8 @@
     "teamstoolkit.guides.cicdPipeline.label": "Automate CI/CD Pipelines",
     "teamstoolkit.guides.mobilePreview.detail": "Run and debug your Teams application on iOS or Android client.",
     "teamstoolkit.guides.mobilePreview.label": "Run and Debug on Mobile Client",
+    "teamstoolkit.guides.multiTenant.detail": "Enable Multi-Tenant support for Teams app.",
+    "teamstoolkit.guides.multiTenant.label": "Multi-Tenant support",
```
Comment (tecton) [inline packages/vscode-extension/package.nls.json:326]:
```suggestion
    "teamstoolkit.guides.multiTenant.label": "Multi-Tenant Support",
```

**Review by tecton — APPROVED** [review]

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -322,6 +322,8 @@
     "teamstoolkit.guides.cicdPipeline.label": "Automate CI/CD Pipelines",
     "teamstoolkit.guides.mobilePreview.detail": "Run and debug your Teams application on iOS or Android client.",
     "teamstoolkit.guides.mobilePreview.label": "Run and Debug on Mobile Client",
+    "teamstoolkit.guides.multiTenant.detail": "Enable Multi-Tenant support for Teams app.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:325]: Multi-Tenant -> multi-tenant

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (KennethBWSong) [reply packages/vscode-extension/package.nls.json]: Updated,thx!

File: packages/vscode-extension/package.nls.json (hunk 2 above)
Comment (KennethBWSong) [reply packages/vscode-extension/package.nls.json]: Updated,thx!

**Review by sffamily — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**